docs/checkit.js

```diff
@@ -732,7 +742,8 @@ CheckIt.prototype.findCurrentDay = function() {
 
 $(document).ready(function() {
     
-    checkit = new CheckIt();
+    checkit = new CheckIt('localStorage');
```

Comment:
Oh I see the problem, this is in `checkit.js`. You probably want this module to have all the code you need to *define* a Checkit, but not actually create one. The code to create the object can either be in a `main.js` or `app.js` or whatever the cool kids are calling their entry point these days, or even just in a `<script>` tag on the main page, that way you can have `mode = 'firebase'` there and `mode = 'localStorage'` in your test html.